template/pyproject.toml.jinja

```diff
@@ -1,16 +1,29 @@
 [build-system]
-{%- if backend == "hatch" %}
 requires = [
+{%- if backend == "hatch" %}
   "hatchling",
+{%- elif backend == "setuptools" %}
+  "setuptools",
+{%- endif %}
 {%- if backend == "hatch" and vcs == true %}
   "hatch-vcs",
+{%- elif backend == "setuptools" and vcs == true %}
+  "setuptools_scm[toml]>=8.1.0",
 {%- endif %}
 ]
+{%- if backend == "hatch" %}
 build-backend = "hatchling.build"
+{%- elif backend == "setuptools" %}
+build-backend = "setuptools.build_meta"
 {%- endif %}
 
 [project]
 name = "{{ project_name }}"
+{%- if backend == "setuptools" and vcs == false %}
+version = "v0.1.0"
```

Comment:
No `v` + a minor version to start with
```suggestion
version = "0.0.1"
```

Reply:
Oops, thanks, I missed that during my review!